sdk/core/core/inc/az_http_transport.h

```diff
@@ -85,12 +85,16 @@ AZ_NODISCARD az_result
 az_http_request_get_header(_az_http_request const* request, int32_t index, az_pair* out_header);
 
 /**
- * @brief Get parts of an HTTP request.
+ * @brief Get parts of an HTTP request. NULL in accepted to ignore getting any parts, for example,
+ * call this function like below to get only the http method and ignore getting url and body
```

Comment:
```suggestion
 * call this function like below to get only the http method and ignore getting url and body.
```

Also consider moving the 2nd sentence with example to a `@note` or `@remarks` instead of making it part of the `@brief`.